Dockerfile

```diff
@@ -22,4 +22,16 @@ LABEL org.opencontainers.image.description="CLI tool and library for generating
 LABEL org.opencontainers.image.vendor="Red Hat, Inc."
 LABEL org.opencontainers.image.licenses="Apache-2.0"
 
+# required per https://github.com/release-engineering/rhtap-ec-policy/blob/main/data/rule_data.yml
+LABEL com.redhat.component="TODO: The Bugzilla component name where bugs against this container should be reported by users."
+LABEL version="0.89.0"
+# TODO: how do we make sure to bump this
+LABEL release="1"
+LABEL name="syft"
+LABEL description="CLI tool and library for generating a Software Bill of Materials from container images and filesystems"
+LABEL io.k8s.description="CLI tool and library for generating a Software Bill of Materials from container images and filesystems"
+LABEL vendor="Red Hat, Inc."
+LABEL url="TODO: A URL where the user can find more information about the image."
```

Comment:
AFAIK this is the catalog page for most RH containers. Can it just be a link to this repo?